src/App.scss

```diff
@@ -1,43 +1,49 @@
-:root {
-  --size-min-window: 800px;
-  --size-max-window: 1296px;
-}
+#root {
+  --layout-window-width-min: 800px;
+  --layout-window-width-max: 1296px;
 
-body {
-  overflow-y: hidden;
-}
+  // TODO: fix this in SDS
+  overflow-x: auto !important;
```

Comment:
Do you have a ticket for this TODO? Are you tracking it somehow?

Reply:
Yes, I have a list where I'm tracking all this. 👍